packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -92,50 +91,72 @@ class DefaultMutationMapperStore implements MutationMapperStore {
-                this.transcriptsWithAnnotations.result.length > 0 &&
-                canonicalTranscriptId &&
-                !this.transcriptsWithAnnotations.result.includes(
-                    canonicalTranscriptId
-                )
-            ) {
-                // if there are annotated transcripts and activeTranscript does
-                // not have any, change the active transcript
-                activeTranscript = this.transcriptsWithAnnotations.result[0];
-            } else {
-                activeTranscript = canonicalTranscriptId;
-            }
-        } else {
-            activeTranscript = canonicalTranscriptId;
-        }
+    readonly activeTranscript: MobxPromise<string | undefined> = remoteData(
+        {
+            await: () => [this.allTranscripts, this.canonicalTranscript],
+            invoke: async () => {
+                let activeTranscript;
+                const canonicalTranscriptId = this.canonicalTranscript.result
+                    ? this.canonicalTranscript.result.transcriptId
+                    : undefined;
 
-        // TODO this.activeTranscript = activeTranscript?
-        return activeTranscript;
-    }
+                const selectedTranscript =
+                    this._selectedTranscript &&
+                    this.allTranscripts.result &&
+                    this.allTranscripts.result
+                        .map(transcript => transcript.transcriptId)
+                        .includes(this._selectedTranscript)
+                        ? this._selectedTranscript
+                        : undefined;
+
+                if (selectedTranscript) {
+                    activeTranscript = selectedTranscript;
+                } else if (this._activeTranscript !== undefined) {
+                    activeTranscript = this._activeTranscript;
+                } else if (this.config.filterMutationsBySelectedTranscript) {
+                    if (
+                        this.transcriptsWithAnnotations.result &&
+                        this.transcriptsWithAnnotations.result.length > 0 &&
+                        canonicalTranscriptId &&
+                        !this.transcriptsWithAnnotations.result.includes(
+                            canonicalTranscriptId
+                        )
+                    ) {
+                        // if there are annotated transcripts and activeTranscript does
+                        // not have any, change the active transcript
+                        activeTranscript = this.transcriptsWithAnnotations
+                            .result[0];
+                    } else {
+                        activeTranscript = canonicalTranscriptId;
+                    }
+                } else {
+                    activeTranscript = canonicalTranscriptId;
+                }
 
-    public set activeTranscript(transcript: string | undefined) {
+                // TODO this.activeTranscript = activeTranscript?
```

Comment:
no need.  this is the one and only place where the app should look to find the active transcript